AGPs/AGP-CS.md

```diff
@@ -0,0 +1,64 @@
+---
+AGP: N/A
+Title: Develop and launch Aragon Chain
+Author: Aidan Hyman & Gregory Markou
+Status: Stage III
+Track: Finance
+Created: 2019-10-03
+---
+
+# Develop and launch Aragon Chain
+
+## Address of the transfer recipient
+
+TBD
+
+## Amount of the transfer
+
+Upon community review, a better estimate can be made based on the needs and requirements from the community.
```

Comment:
This section will need to be completed with a final amount before the Stage V deadline.